src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -95,23 +95,31 @@ Once configured, all events will be sent to the `/tunnel` endpoint. This solutio
 
 ```php
 <?php
+// Change $host appropriately if you run your own Sentry instance.
+$host = "sentry.io";
+// Set $known_project_ids to an array with your Sentry project IDs which you
+// want to accept through this proxy.
+$known_project_ids = array(  );
+
 $envelope = stream_get_contents(STDIN);
 $pieces = explode("\n", $envelope, 2);
 $header = json_decode($pieces[0], true);
 if (isset($header["dsn"])) {
     $dsn = parse_url($header["dsn"]);
     $project_id = intval(trim($dsn["path"], "/"));
-    $options = array(
+    if (in_array($project_id, $known_project_ids)) {
+      $options = array(
         'http' => array(
-            'header'  => "Content-type: application/octet-stream\r\n",
+            'header'  => "Content-type: application/x-sentry-envelope\r\n",
```

Comment:
Updated based on https://develop.sentry.dev/sdk/envelopes/#http-headers

I don't know why we had `application/octet-stream` here before. We could either ignore this completely for simplicity (as Relay doesn't enforce it), or use `application/x-sentry-envelope`.